skorch/net.py

```diff
@@ -567,56 +540,214 @@ def _apply_virtual_params(self, virtual_kwargs):
     def initialize_virtual_params(self):
         self.virtual_params_ = {}
 
-    def initialize_optimizer(self, triggered_directly=True):
+    def initialize_optimizer(self):
```

Comment:
Changes public API, but should be okay?

Reply:
Indeed. In my memory, this argument was just added recently and thus I thought it's unlikely that anyone uses it. But it's actually been there for 2.5 years, so I will add a deprecation.